File: CS/MultiSelectColumnCustomization/CustomGridControl/GridViewOverride.cs
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Views.Grid;
using DevExpress.XtraGrid.Views.Grid.Customization;

namespace MultiSelectColumnCustomization
{
	public class GridViewOverride : GridView
	{
		public GridViewOverride(GridControl ownerGrid)
			: base(ownerGrid)
		{
		}
		public GridViewOverride()
		{
		}

		protected override CustomizationForm CreateCustomizationForm()
		{
			return new MultiSelectCustomizationForm(this);
		}

		protected override string ViewName
		{
			get
			{
				return "GridViewOverride";
			}
		}
	}
}

File: VB/MultiSelectColumnCustomization/CustomGridControl/GridViewOverride.vb
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Views.Grid
Imports DevExpress.XtraGrid.Views.Grid.Customization

Namespace MultiSelectColumnCustomization

    Public Class GridViewOverride
        Inherits GridView

        Public Sub New(ByVal ownerGrid As GridControl)
            MyBase.New(ownerGrid)
        End Sub

        Public Sub New()
        End Sub

        Protected Overrides Function CreateCustomizationForm() As CustomizationForm
            Return New MultiSelectCustomizationForm(Me)
        End Function

        Protected Overrides ReadOnly Property ViewName As String
            Get
                Return "GridViewOverride"
            End Get
        End Property
    End Class
End Namespace
